packages/xchain-aggregator/src/protocols/chainflip/chainflipProtocol.ts

```
@@ -285,4 +226,127 @@ export class ChainflipProtocol implements IProtocol {
+    const networkFee = actualQuote.includedFees.find((fee) => fee.type === 'NETWORK')
+    const boostFee = actualQuote.includedFees.find((fee) => fee.type === 'BOOST')
+
+    const baseWarning =
+      'Do not cache this response. Open a deposit channel immediately before broadcast; do not send funds after channel expiry.'
+
+    return {
+      protocol: this.name,
+      // Quote-only: channel is opened via openDepositChannel / doSwap
+      toAddress: '',
+      memo: '',
+      expectedAmount: new CryptoAmount(baseAmount(actualQuote.egressAmount, destAssetData.decimals), params.destinationAsset),
```

ESLint

[error] Replace `baseAmount(actualQuote.egressAmount,·destAssetData.decimals),·params.destinationAsset` with `⏎········baseAmount(actualQuote.egressAmount,·destAssetData.decimals),⏎········params.destinationAsset,⏎······`

(prettier/prettier)